cg/apps/demultiplex/sample_sheet/models.py

```diff
@@ -34,7 +34,6 @@ class SampleDragen(NovaSeqSample):
 
 
 class SampleSheet(BaseModel):
-    flow_cell_mode: str
     samples: List[NovaSeqSample]
 
     class Config:
```

Comment:
I think we can remove this as well.

Reply:
Great catch!

Reply:
Or you mean the whole class?

Reply:
Just the whole class Config. No need to keep it as the enum evaluation inn flow cell mode is gone